static/images/badges/es/snap-store-black.svg

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="182px" height="56px" viewBox="0 0 182 56" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 52.3 (67297) - http://www.bohemiancoding.com/sketch -->
+    <!-- Generator: Sketch 52.4 (67378) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Probably not necessary in this PR (as likely affects all the badges), but seems like these are straight from Sketch and could be optimized with some tool like SVGOMG.